USAGE.rst

```diff
@@ -46,19 +46,20 @@ Step-by-Step
    # switch to your input directory
    cd $HOME/picInputs/myLWFA
 
-PIConGPU is controlled via two kinds of input sets: compile-time options and runtime options.
+PIConGPU is controlled via two kinds of textual input sets: compile-time options and runtime options.
 
-Edit the :ref:`.param files <usage-params>` inside ``include/picongpu/param/``.
-Initially and when options are changed, PIConGPU *requires a re-compile*.
+Compile-time :ref:`.param files <usage-params>` reside in ``include/picongpu/param/`` and define the physics case and deployed numerics.
+After creation and whenever options are changed, PIConGPU *requires a re-compile*.
+Feel free to take a look now, but we will later come back on how to :ref:`edit those files <usage-params-edit>`.
 
-Now edit the :ref:`runtime (command line) arguments <usage-cfg>` in ``etc/picongpu/*.cfg``.
-These options do *not* require a re-compile when changed (e.g. simulation size, number of GPUs, plugins, ...).
+:ref:`Runtime (command line) arguments <usage-cfg>` are set in ``etc/picongpu/*.cfg`` files.
+These options do *not* require a re-compile when changed (e.g. simulation size, number of devices, plugins, ...).
 
 2. Compile Simulation
 """""""""""""""""""""
 
-In our input, ``.param`` files are build directly into the PIConGPU binary for performance reasons.
-Changing or initially adding those requires a compile.
+In our input, ``.param`` files are build directly into the PIConGPU binary for :ref:`performance reasons <usage-params-rationale>`.
+A compile is required after changing or initially adding those files.
```

Comment:
This is a repetition of the content of line 51-53.

Reply:
See long message below. You reviewed and merged this already in #2685.
If you have further suggestions now, please open a PR with your proposed changes to `dev`.

This backport, cherry-pick only PR is not the right place for this.